ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-rhacs-docs-4.3.yaml

```
@@ -0,0 +1,36 @@
+build_root:
+  image_stream_tag:
+    name: openshift-docs-asciidoc
+    namespace: ocp
+    tag: latest
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 200m
+      memory: 400Mi
+tests:
+- as: deploy-preview
+  steps:
+    env:
+      DISTROS: openshift-acs
+      PREVIEW_SITE: ocpdocs-pr
+    test:
+    - ref: openshift-docs-build-docs
+    - ref: openshift-docs-preview-comment
```

```suggestion
    - ref: openshift-docs-preview-comment-pages
```
If you want to add the updated URLs to the preview comment. See https://github.com/openshift/release/pull/50353